app/models/post.rb

```diff
@@ -1,5 +1,5 @@
 class Post < ApplicationRecord
-  belongs_to :author, class_name: 'User'
+  belongs_to :user, class_name: 'User'
```

Comment:
- [ ] Your code says `Post` belongs to `user` but your migration file for creating `Posts` references `author_id`. I would advise you to reference `author` in your migration and then in your model file, `Post` should belong to `author` instead of `user`. I won't be able to create a post if this change is not made and that is the reason why I am making it a required change. 👍